aws/tests.py

```diff
@@ -0,0 +1,76 @@
+"""
```

Comment:
Please make a tests folder and move the tests there, to be consistent with the jenkins part of the tree and to be poised for when we inevitable glom more features onto this repo. :)